cloudigrade/api/migrations/0038_instancedefinition.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 3.1.7 on 2021-03-02 15:40
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0037_delete_instance_definition"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
Since the table will be empty immediately following this deployment, if there are any concurrent usage calculation jobs that run shortly after this deployment, they will likely produce incorrect totals.

I think we need to make AWS calls to rebuild the definitions here in this migration or run `loadec2definitions` after `migrate` in the deployment mid-hook. Calling within the migration could be tricky since the model classes are in a weird state and you can't easily import and call our functions. The latter feels bad to introduce AWS as a dependency during all deployments, but it's much easier to implement and we could pull it back out after one deploy. Thoughts or other ideas?

Reply:
[Thread](https://ansible.slack.com/archives/C8VGAPJNN/p1614712944056100?thread_ts=1614710040.053600&cid=C8VGAPJNN)